### ISSUE TITLE
We need Red Alert 2 Remastered! We need Red Alert 2 Yuri's Revenge Remastered!

### ISSUE THREAD

**Issue by txdtplus** [electronicarts/CnC_Remastered_Collection#111]
In China, Red Alert 2 is much much much more famous than C&C 1 and RA1. RA2 is also hot even in 2021. However, only a few Chinese people can acquire the original game. I am sorry that I played the pirated game in the 2000s. I even didn't know that is pirated game.

I love RA2 and Yuri's Revenge, I am really want to pay for the legitimate game. Please remake Red Alert 2 Remastered and Red Alert 2 Yuri's Revenge Remastered! We will pay for it！

**Comment by DanielTongAwesome** [comment]
yes, that is true, even people pay it nowadays ~ btw, I hope there is a RA2 mac version and that is the main reason why I am here

**Comment by DanielTongAwesome** [comment]
I heard that RA2's source code was lost that is why people are rebuilding it nowadays? I would really like to contribute to build a native mac version RA2 Yuri's Revenge ~ btw, in China, not many people actually pay the RA2 since its so easy to find a free download on Baidu.com

**Comment by txdtplus** [comment]
> I heard that RA2's source code was lost that is why people are rebuilding it nowadays? I would really like to contribute to build a native mac version RA2 Yuri's Revenge ~ btw, in China, not many people actually pay the RA2 since its so easy to find a free download on Baidu.com

You are right. But RA2 will be stuck when there are too many buildings and tanks because RA2 only supports a single core CPU.  If EA makes RA2 support multicore CPU and GPU. I believe that more people will pay for it!

**Comment by DanielTongAwesome** [comment]
emmmm .... so they actually lost the source code ... then was it possible to rebuild one?

**Comment by txdtplus** [comment]
If they actually lost the source code, that is so regrettable...

**Comment by panxinfeng** [comment]
> > I heard that RA2's source code was lost that is why people are rebuilding it nowadays? I would really like to contribute to build a native mac version RA2 Yuri's Revenge ~ btw, in China, not many people actually pay the RA2 since its so easy to find a free download on Baidu.com
> 
> You are right. But RA2 will be stuck when there are too many buildings and tanks because RA2 only supports a single core CPU. If EA makes RA2 support multicore CPU and GPU. I believe that more people will pay for it!

thank for your answer that i can learn more about ra2.

**Comment by rafnogueira** [comment]
I really agree, I played RA2s so much when I was kid. I definitively will support this

**Comment by JamiePlur** [comment]
can't agree more
